libs/pilight/protocols/433.92/bresser_3ch.c

```diff
@@ -0,0 +1,277 @@
+#include <stdio.h>
```

Comment:
The MPLv2 license header is missing

Reply:
This is not how it's done on the other code already in MPLv2. E.g. check `kerui_d026.c`

Reply:
Also, a lot of newlines between comment and code blocks are missings.